data/json/items/gunmod/sights.json

```diff
@@ -317,5 +317,24 @@
     "consume_divisor": 85,
     "min_skills": [ [ "weapon", 2 ], [ "gun", 3 ] ],
     "flags": [ "DISABLE_SIGHTS", "CONSUMABLE", "ZOOM" ]
+  },
+  {
+    "id": "type99_scope",
+    "type": "GUNMOD",
+    "copy-from": "rifle_scope",
+    "name": { "str": "Nagoya 99 gun scope" },
+    "//": "Kokura also made and issued 2.5x rifle scopes for the Type 99, but those are vanishingly rare when compared against Nagoya-built, non-adjustable 4x scopes. That being said, Nagoya also built a small number of 2.5x scopes and dabbled with an adjustable 4x close to the end of production, but those are also comparatively rare enough to be hand-waved away.",
+    "description": "A basic and antique 4x rifle optic: essentially a non-adjustable telescopic sight built by the Nagoya Arsenal during the Second World War and issued for mounting on Type 99 sniper rifles.",
+    "range_multiplier": 3,
```

Comment:
Shouldn't this have a different range multipler than the x2.5 scope 🤔

Reply:
According to my reading, the effective range modifiers were around a factor of three for both scopes: 1500 metres for a Type 97 with a telescopic sight as opposed to 500 metres with a normal 38, and 1500 metres for a Type 99 with a gun sight versus 657 metres for regular iron sights. Considering the numbers, I suppose the 4x could be nocked down to a 2-times modifier, or we could just calculate the distance in tiles when factoring range compression and work from there. As for the other two statistics, I didn’t particularly understand how the aim speed modifier affected the weapon from a game mechanics standpoint, and I couldn’t tell if I should tweak the field of view numbers.